governance/remote_executor/programs/remote-executor/src/tests/executor_simulator.rs

```diff
@@ -81,9 +83,12 @@ pub enum VaaAttack {
 impl ExecutorBench {
     /// Deploys the executor program as upgradable
     pub fn new() -> ExecutorBench {
-        let mut bpf_data = read_file(find_file("remote_executor.so").unwrap_or_else(|| {
-            panic!("Unable to locate {}", "remote_executor.so");
-        }));
+        let mut bpf_data = read_file(
+            std::env::current_dir()
+                .unwrap()
+                .join(Path::new("../../target/deploy/remote_executor.so")),
+        );
```

Comment:
Newer versions of solana can't find the binary with the previous logic

Reply:
Can you explain why?

Reply:
Previous versions "1.13.x" would set an environment variable BPF_OUT_DIR and that's how the code would find the binary.
New versions "1.14.x" don't seems to set it, so `find_file` fails.

Reply:
Yeah I recall the `BPF_OUT_DIR` issues as well. Can you mention what might be the reason it's not doing it before we merge this? Don't worry if it takes too much time you can merge over this commit, it would just be nice to know what changed here for other solana related work.